### ISSUE TITLE
command injection vulnerability - noted from npm audit

### ISSUE THREAD

**Issue by jeremykorb** [jsoverson/grunt-open#35]
`npm audit` barks an alert to a critical vulnerability with this package. 
 It returns:

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Critical      │ Command Injection                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ open                                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ No patch available                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ grunt-open [dev]                                             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ grunt-open > open                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/663                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

**Comment by mitraparthib3** [comment]
I have seen the same issue, looks like package `open` is not under proper maintenance.

**Comment by Spongman** [comment]
`grunt-open` should switch to using https://github.com/sindresorhus/opn instead.

**Comment by jsoverson** [comment]
Resolved with 0.2.4
